instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry'
+
+module OpenTelemetry
+  module Instrumentation
+    module GraphQL
+      module Tracers
+        # GraphQLTrace contains the OpenTelemetry tracer implementation compatible with
+        # the new GraphQL tracing API (>= 2.0.18)
+        module GraphQLTrace # rubocop:disable Metrics/ModuleLength
```

Comment:
"Tracer" vs "Trace" is confusing but that's following the graphql gem's convention.